bot/seasons/evergreen/issues.py

```diff
@@ -3,55 +3,74 @@
+              403: "rate limit has been hit! Please try again later!"}
+
 
 class Issues(commands.Cog):
     """Cog that allows users to retrieve issues from GitHub."""
 
     def __init__(self, bot: commands.Bot):
         self.bot = bot
 
-    @commands.command(aliases=("issues",))
+    @commands.command(aliases=("pr",))
     @override_in_channel()
-    async def issue(
-        self, ctx: commands.Context, number: int, repository: str = "seasonalbot", user: str = "python-discord"
-    ) -> None:
-        """Command to retrieve issues from a GitHub repository."""
-        api_url = f"https://api.github.com/repos/{user}/{repository}/issues/{number}"
-        failed_status = {
-            404: f"Issue #{number} doesn't exist in the repository {user}/{repository}.",
-            403: f"Rate limit exceeded. Please wait a while before trying again!"
-        }
-
-        async with self.bot.http_session.get(api_url) as r:
-            json_data = await r.json()
-            response_code = r.status
+    async def issue(self, ctx: commands.Context, number: int, repository: str = "seasonalbot",
+                    user: str = "python-discord") -> None:
+        """Command to get issues/pull request from GitHub."""
+        url = f"https://api.github.com/repos/{user}/{repository}/issues/{str(number)}"
+        mergeURL = f"https://api.github.com/repos/{user}/{repository}/pulls/{str(number)}/merge"
 
-        if response_code in failed_status:
-            return await ctx.send(failed_status[response_code])
-
-        repo_url = f"https://github.com/{user}/{repository}"
-        issue_embed = discord.Embed(colour=Colours.bright_green)
-        issue_embed.add_field(name="Repository", value=f"[{user}/{repository}]({repo_url})", inline=False)
-        issue_embed.add_field(name="Issue Number", value=f"#{number}", inline=False)
-        issue_embed.add_field(name="Status", value=json_data["state"].title())
-        issue_embed.add_field(name="Link", value=json_data["html_url"], inline=False)
-
-        description = json_data["body"]
-        if len(description) > 1024:
-            placeholder = " [...]"
-            description = f"{description[:1024 - len(placeholder)]}{placeholder}"
+        async with self.bot.http_session.get(url) as r:
+            json_data = await r.json()
 
-        issue_embed.add_field(name="Description", value=description, inline=False)
+        if r.status in RESP_VALUE:
+            return await ctx.send(f"[{str(r.status)}] {RESP_VALUE.get(r.status)}")
+        # the original call is made to the issues API endpoint
+        # if a issue or PR exists then there will be something returned
+        # if the word 'issues' is present within the response then we can simply pull the  data we need from the
+        # return data received from the API
+        if "issues" in json_data.get("html_url"):
+            if json_data.get("state") == "open":
+                iconURL = ICONS.get("ISSUE")
```

Comment:
I would prefer adhering to PEP8 for the variable name, eg. icon_URL or icon_url.